matomo.php

```diff
@@ -145,4 +169,63 @@ public function onTwigTemplatePaths()
     {
         $this->grav['twig']->twig_paths[] = __DIR__ . '/templates';
     }
+
+    /**
+     * Documents the reason for blocking Matomo tracking in a JavaScript comment
+     * @param string $reason
+     */
+    private function documentBlockingReason(string $reason)
+    {
+        if ($this->config->get('plugins.matomo.debug', false))
+            $this->grav['assets']->addInlineJs("/* Matomo tracking blocked, reason: $reason */");
```

Comment:
Oh an please, always add brackets around ifs. It will prevent you from so many errors.

Reply:
Not really an issue for me, maybe because I've always been using some form of auto-indenting editor or IDE for brace-style languages. But within one code base, consistency is always preferable, so next time I'll hopefully adapt better.